web_external/ImagesGallery/Detail/imageDetailsPage.jade

```diff
@@ -86,9 +86,10 @@
       tr
         td.isic-image-details-table-section-header(colspan=2) Tags
       tr
-        td
+        td(colspan=2)
           each value in image.get('notes').tags
-            span.label.label-default= value
-        td
+            //- Labels separated by a space
+            span.label.label-default.isic-image-details-tag= value
+            | &#x20;
```

Comment:
- Normally, yes, I'd first reach for `&nbsp;`. But here the intent of the space is to actually break the line, in addition to providing a visual buffer between the labels.
- The space is outside of the span, so it shouldn't be on the same line as the value.

I'll rebase/squash now.